content/about/privacy.md

```diff
@@ -0,0 +1,96 @@
+
+This privacy policy will explain how our organisation uses the personal data we collect from you when you use our website and contribute to our projects.
+
+## What Data Do We Collect And Why?
+
+You directly provide InnerSource Commons with most of the data we collect. We collect and process data when you:
+
+- Register to our emailing list: InnerSource Commons collects personal identification information (Name and Email Address) to email you with news, events and articles we think you might like.
+
+- Register to our events: InnerSource Commons collects information that you provide us when you register for one of our events, such as conferences and when you choose to participate in an event as an attendee, speaker, or sponsor. This information includes personal data such as your name, company, and contact information.
+
+- Join our Slack community: When you join the conversation on Slack we store the information you contribute and your conversations.
+
+- Voluntarily contribute to our GitHub Repositories by providing feedback, content or committing code: When you contribute source code, documentation, or other materials to one of our projects (whether on your own behalf or on behalf of your employer), we store the information and content that you contribute. This can include the contributed materials, information required to confirm the provenance of the contribution, and any associated intellectual property. We ask you not to upload personal data to our projects.
+
+- Voluntarily answer questionnaires and surveys: From time to time we may send you questionnaires or surveys for collecting demographic information or assessing your interests and needs, to help us to improve our services. Any such questionnaires or surveys will be entirely voluntary. The purpose and intended use of the information being collected will be explained in the survey itself.
+
+### Information Automatically Collected From You
+
+We also collect certain information automatically. Like many websites, the InnerSource Commons  website uses “cookies” and we obtain certain types of information when your web browser accesses our site. On the InnerSource Commons website Cookies are used for the following purposes and using the following services:
+ 
+Analytics
+
+- Google Analytics - Personal Data: Cookies and Usage Data. We track number of visitors per page on our website.
+
+Displaying content from external platforms
+
+- Twitter - Personal Data: Usage Data and various types of Data as specified in the [privacy policy](https://twitter.com/en/privacy) of the service
+- Youtube - Personal Data: Usage Data and various types of Data as specified in the [privacy policy](https://policies.google.com/privacy) of the service
+
+Forms
+
+- MailChimp (emailing list) - Personal Data: Cookies and various types of Data as specified in the [privacy policy](https://mailchimp.com/legal/privacy/) of the service
```

Comment:
The current form on the website not generates any cookies. All data used in the subscription form (currently this data is the only email) sent to the Mailchimp servers directly after the user clicks on a "Subscribe" button.

Reply:
After we click subscribe, on the page that the user is brought to are the following cookies:

![Screenshot 2021-03-03 at 14 21 46](https://user-images.githubusercontent.com/78749459/109820517-fbcf3c80-7c2c-11eb-8ee6-4df7ed2bb619.png)
![Screenshot 2021-03-03 at 14 21 55](https://user-images.githubusercontent.com/78749459/109820531-ff62c380-7c2c-11eb-9eaf-f1ba144c14da.png)

Reply:
> After we click subscribe, on the page that the user is brought to are the following cookies:

After the click, we move to a 3-rd party site with its own domain, cookies and policies. is not it so?

Reply:
You are right it is a 3rd party site but I think we are still responsible for it. All we are saying is that we use Mailchimp for the service and that they use cookies. So if you are interested in their cookies policy you can find it there. It would be great to get someone else's perspective on this too.

Reply:
I'm not sure about our responsibility for cookies on 3rd party sites: if it really is -- then we should scan through any external links and check their cookies :)

Reply:
Can we just copy the language from another privacy policy on a site who we know uses Mailchimp too?